spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -288,6 +289,31 @@ def test_get_namespace_view(kernel):
     assert "'python_type': 'int'" in nsview
 
 
+@pytest.mark.parametrize("filter_on", [True, False])
+def test_get_namespace_view_filter_on(kernel, filter_on):
+    """
+    Test the namespace view of the kernel with filters on and off.
+    """
+    execute = asyncio.run(kernel.do_execute('TestFilterOn = 1', True))
+    asyncio.run(kernel.do_execute('TestFilterOff = 1', True))
```

Comment:
Not totally sure but we just need one of this variables, right? Other idea could be to define one that variable that always is shown regardelss of the filters (executing something like `a = 1`) and another one that is only shown when the filters are off (it could be `TestFilterOff`). What do you think @jsbautista ?

Reply:
> 

You are right, I agree, I will make the necessary changes